Library/Formula/mono.rb

```diff
@@ -30,7 +34,13 @@ def install
       --prefix=#{prefix}
       --enable-nls=no
     ]
-    args << "--build=" + (MacOS.prefer_64_bit? ? "x86_64": "i686") + "-apple-darwin"
+
+    if build.with? "64bit"
+      args << "--build=x86_64-apple-darwin"
+    else
+      # no option specified
+      args << "--build=" + (MacOS.prefer_64_bit? ? "x86_64": "i686") + "-apple-darwin"
```

Comment:
Look in the history of this formula. If you want to me to change it which might break other formulae that use mono, just tell me to go ahead, and I'll do it.